python/pyarrow/parquet/__init__.py

```diff
@@ -2928,8 +2928,10 @@ def write_to_dataset(table, root_path, partition_cols=None,
                      partition_filename_cb=None, filesystem=None,
                      use_legacy_dataset=None, schema=None,
                      partitioning=None, basename_template=None,
-                     use_threads=None, file_visitor=None, **kwargs):
-    """Wrapper around parquet.write_dataset for writing a Table to
+                     use_threads=None, file_visitor=None,
+                     existing_data_behavior='overwrite_or_ignore',
```

Comment:
Set the default to `overwrite_or_ignore` because of the specification of `basename_template`.

Reply:
Changed it to None.